spark-tpcds-benchmark-runner/src/main/java/com/palantir/spark/benchmark/config/BenchmarksConfiguration.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.spark.benchmark.config;
+
+import static com.palantir.logsafe.Preconditions.checkArgument;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.palantir.spark.benchmark.immutables.ImmutablesConfigStyle;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@ImmutablesConfigStyle
+@JsonDeserialize(as = ImmutableBenchmarksConfiguration.class)
+public interface BenchmarksConfiguration {
+    int iterations();
```

Comment:
would be good to have the ability to flag on/off tpcds and sort benchmarks separately in this config

Reply:
parallel to `SourceDataGenerationConfig` fields in the `DataGenerationConfiguration` object